src/utils/components/PolicyForm/PolicyWizard/ReviewStep.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { FC } from 'react';
+import { NodeModelGroupVersionKind } from 'src/console-models/NodeModel';
+
+import { IoK8sApiCoreV1Node } from '@kubevirt-ui/kubevirt-api/kubernetes/models';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  Alert,
+  AlertVariant,
+  Content,
+  Form,
+  Level,
+  LevelItem,
+  Stack,
+  StackItem,
+  Title,
+} from '@patternfly/react-core';
+import {
+  InterfaceType,
+  NodeNetworkConfigurationInterface,
+  V1NodeNetworkConfigurationPolicy,
+} from '@types';
+import { NODE_HOSTNAME_LABEL } from '@utils/constants';
+import { useNMStateTranslation } from '@utils/hooks/useNMStateTranslation';
+import { filterPolicyAppliedNodes } from '@utils/policies/utils';
+
+import InfoBox from './components/InfoBox';
+import InterfaceReview from './components/InterfaceReview';
+
+type ReviewStepProps = {
+  policy: V1NodeNetworkConfigurationPolicy;
+  error?: Error;
+};
+
+const ReviewStep: FC<ReviewStepProps> = ({ policy, error }) => {
+  const { t } = useNMStateTranslation();
+
+  const ethernetInterfaces: NodeNetworkConfigurationInterface[] =
+    policy.spec?.desiredState?.interfaces?.filter(
+      (iface) => iface.type === InterfaceType.ETHERNET,
+    ) || [];
+  const bondingInterfaces: NodeNetworkConfigurationInterface[] =
+    policy.spec?.desiredState?.interfaces?.filter((iface) => iface.type === InterfaceType.BOND) ||
+    [];
+  const bridgeInterfaces: NodeNetworkConfigurationInterface[] =
+    policy.spec?.desiredState?.interfaces?.filter((iface) =>
+      [InterfaceType.LINUX_BRIDGE, InterfaceType.OVS_BRIDGE].includes(iface.type),
+    ) || [];
+
+  const [nodes] = useK8sWatchResource<IoK8sApiCoreV1Node[]>({
+    groupVersionKind: NodeModelGroupVersionKind,
+    isList: true,
+  });
+
+  const qualifiedNodes = filterPolicyAppliedNodes(nodes, policy);
+
+  const qualifiedNodeHostnames = qualifiedNodes.map(
+    (node) => node.metadata?.labels?.[NODE_HOSTNAME_LABEL],
```

Comment:
Do you have selectors for common `K8sResourceCommon` properties (like `getLabels`) in this repo?

Reply:
No but i'll add them